react-native-libraries.json

```
@@ -11902,5 +11902,26 @@
     "android": true,
     "web": true,
     "expoGo": true
+  },
+  {
+    "githubUrl": "https://github.com/PostHog/posthog-js-lite/tree/main/posthog-react-native",
+    "npmPkg": "posthog-react-native",
+    "examples": [
+      "https://github.com/PostHog/posthog-js-lite/tree/main/examples/example-expo",
+      "https://github.com/PostHog/posthog-js-lite/tree/main/examples/example_rn"
+    ],
+    "images": ["https://posthog.com/static/a31f7db2abf56ba7e82e88b0d47ea0be/929d0/screenshot-product-analytics.webp"],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "windows": false,
+    "macos": true,
+    "tvos": false,
+    "visionos": false,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
+    "newArchitecture": true
```

```suggestion
    "githubUrl": "https://github.com/PostHog/posthog-js-lite/tree/main/posthog-react-native",
    "examples": [
      "https://github.com/PostHog/posthog-js-lite/tree/main/examples/example-expo",
      "https://github.com/PostHog/posthog-js-lite/tree/main/examples/example_rn_new_architecture",
      "https://github.com/PostHog/posthog-js-lite/tree/main/examples/example_rn_macos"
    ],
    "images": ["https://posthog.com/static/a31f7db2abf56ba7e82e88b0d47ea0be/929d0/screenshot-product-analytics.webp"],
    "ios": true,
    "android": true,
    "web": true,
    "expoGo": true,
    "macos": true,
    "newArchitecture": true
```

Entry cleanup suggestion.

For marking out-of-tree platforms as supported we require an example, and I as able to find one in your repository, so let's link that too.

Also, wdyt about promoting New Architecture example instead of legacy one?

---

I agree, let's do that! 
I'll also add a link to the Expo RN Web example